locales/en-US.json

```
@@ -152,18 +154,18 @@
   "movies": "Movies",
   "name": "Name",
   "networks": "Networks",
+  "nextEpisodePlayingIn": "Next Episode Playing in",
```

```suggestion
  "nextEpisodePlayingIn": "Next episode playing in {time} seconds",
```

And remember to change the occurrences in the code.

---

I don't think this would work as we want the `{time} seconds` part to be in a different color than the rest

---

Oh, true. Fair enough.